backend/src/Database.hs

```diff
@@ -153,6 +156,12 @@ repsertPlayerStats (totalStats@(PlayerStatsTotal {..}), yearStats@(PlayerStatsYe
   repsert (PlayerStatsTotalKey playerStatsTotalPlayerId) totalStats
   repsert (PlayerStatsYearKey playerStatsYearPlayerId playerStatsYearYear) yearStats
 
+updatePlayerName :: (MonadIO m, MonadLogger m) => PlayerId -> PlayerName -> DBAction m ()
+updatePlayerName pid name = lift $ do
+  update $ \player -> do
+    set player [PlayerName =. val (normalizeName name), PlayerDisplayName =. val name]
```

Comment:
Because of the uniqueness guards you have, will this throw an error if the name is already taken? I can make sure the frontend handles that specific error with a nice message

Reply:
Yes, but I don't actually know what error it will throw! Should figure that out, and maybe do some explicit error handling. Thanks, I'll take a look